1-exercises/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,25 @@
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+  arr.forEach(function(name,index){
+        for(let index=1; index<=5;index++){
+          arr.splice((index*3-1),1,`'Fizz'`);
+        } ;
+        for(let index=2; index<=3;index++){
+          arr.splice((index*index),1,`'Buzz'`);
+       };
+          arr.splice((arr.length-1),1,`'FizzBuzz'`);
+        
+       console.log(name)
+  })
```

Comment:
While this function works for the given array, if the array includes an extra number, e.g., up to 16 then it would print FizzBuzz for 16 instead of for 15 which is not the expected behaviour. I think it would be simpler to write the function only as a function of index and then just write a number of if statements to check if the index is multiple of 3, 5 or both and then print the right option for each case. In that way, the function would be more general.